src/service/owners-impl.js

```diff
@@ -39,7 +67,13 @@ export class Owners {
    * @param {!Element} parentElement
    * @param {!Element|!Array<!Element>} subElements
    */
-  schedulePreload(parentElement, subElements) {}
+  schedulePreload(parentElement, subElements) {
```

Comment:
should leave this as an interface class, and create another real impl class implementing it.